src/scripts/main.js

```diff
@@ -1,3 +1,15 @@
 'use strict';
 
-// write code here
+function sortList(list, fatherSelector) {
+  list.sort((a, b) => b.dataset.salary.slice(1).split(',').join('')
+    - a.dataset.salary.slice(1).split(',').join(''));
+
+  for (const person of list) {
+    document.querySelector(fatherSelector).append(person);
+  }
+}
+
+const listSelector = 'ul';
+const people = [...document.querySelector(listSelector).children];
+
+sortList(people, listSelector);
```

Comment:
and where's getEmployees function?
> Create function getEmployees, which takes a list property, and returns an array of right objects.

Reply:
I think it was only recomendations, but I found better way